tests/integration/synthetics/test_happy_path_with_dt.php

```diff
@@ -62,6 +62,7 @@
         "nr.syntheticsJobId": "jjjjjjj-jjjj-1234-jjjj-jjjjjjjjjjjj",
         "nr.syntheticsMonitorId": "mmmmmmm-mmmm-1234-mmmm-mmmmmmmmmmmm",
         "externalDuration": "??",
+        "externalCallCount": 1,
```

Comment:
Where does this extra attribute come from?

Reply:
It was an external segment bug.
The call count should have been calculated in 2 different places, but it was only updated in one place.
See commit and commit message for info: 
https://github.com/newrelic/newrelic-php-agent/pull/991/commits/6932595b740cf04e4c277e103b6f4748ef932de9